packages/frontend/src/lib/conversation/ChatMessage.svelte

```diff
@@ -35,7 +35,7 @@ function getMessageParagraphs(message: ChatMessage): string[] {
     class:bg-[var(--pd-content-card-inset-bg)]={isAssistantChat(message)}
     class:ml-8={isAssistantChat(message)}
     class:mr-8={isUserChat(message)}>
-    {#each getMessageParagraphs(message) as paragraph (paragraph)}
+    {#each getMessageParagraphs(message) as paragraph, idx (`${idx}-paragraph`)}
```

Comment:
```suggestion
    {#each getMessageParagraphs(message) as paragraph, index (index)}
```

Probably just use the index as key (we should not introduce new name such as idx, we usually use `index`).

Reply:
:flushed: there was a typo there, original intention was to have `${idx}-${paragraph}` to not diverge from the original.

Anyway, I've updated as per your suggestions.